hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileAppendHandlerTest.java

```diff
@@ -143,7 +168,7 @@ void preHandleWorksAsExpected() throws PreCheckException {
 
         subject.preHandle(realPreContext);
 
-        assertTrue(realPreContext.requiredNonPayerKeys().size() > 0);
+        assertFalse(realPreContext.requiredNonPayerKeys().isEmpty());
```

Comment:
Since we're changing tests anyway it might be good to replace junit with assertj on the lines that are changed or added. Just a thought though

Reply:
@mhess-swl I would ask during the call because it is all over the place